models/google_ads__account_report.sql

```diff
@@ -0,0 +1,37 @@
+with stats as (
+
+    select *
+    from {{ var('account_stats') }}
+
+), accounts as (
```

Comment:
style-wise, I think these CTEs should look like (and similar in other models):

```
with stats as (

    select *
    from {{ var('account_stats') }}

), 

accounts as (
```

Reply:
Style updated across models.